api_v2/src/overtime/overtime.service.ts

```
@@ -48,4 +57,35 @@ export class OvertimeService {
       },
     });
   }
+
+  async approveDisapproveOvertime(input: ApproveOvertimeRequestInput): Promise<boolean> {
+    const { userId, overtimeId, approvedMinutes, isApproved, managerRemarks } = input;
+
+    if (!overtimeId) {
+      throw new NotFoundException('Invalid overtime ID.');
+    }
+
+    const overtime = await this.overtimeRepository.findById(overtimeId);
+    if (!overtime) {
+      throw new NotFoundException(`Overtime with ID ${overtimeId} not found.`);
+    }
```

Please hide the exception related to repository. Put it inside findById